src/pages/library/dataBrowser-utils.js

```diff
@@ -56,6 +56,12 @@ export const isDatarepoSnapshot = dataset => {
   return _.toLower(dataset['dcat:accessURL']).includes(datarepoSnapshotUrlFragment)
 }
 
+export const renderConsortium = dataset => _.flow(
+  _.map(dataCollection => dataCollection['dct:title']),
```

Comment:
I think its a little more complicated by that. In this case, nothing in our schema/dataset knows that this is where consortium is pulled from. That actually feels like an oversight, in my book. Because there are two fields, it would be weird to me to say that it is rendered solely by pulling out `'dct:title'`